### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by BigCityCat** [LawnchairLauncher/lawnchair#4231] [General]
I understand that the developers have a lot on the table. I love what has already been accomplished. I also don't have the skills to implement it myself. Having said that. 

What I would like to see is for the status bar to hide entirely and not the current fade in and out. Sometimes it doesn't fade in and out plus it just doesn't look as good. It would be better if it was just hiden from the start. If not then, it would be cool to have the ability to recolor the font on the status bar. 

Thanks for all your work.

**Comment by BigCityCat** [comment]
The Android Status bar sucks is what I was trying to say. If you look at the Play Store there's no app that can help. If you Google hide the status bar you will see that everyone wants it and there's no working solution.

**Comment by whoisabdi** [comment]
I don't think Android launchers have that capability to disable something like the Status Bar entirely. 3rd party launchers are meant to be replacement for your homescreen and the most stuff they can do will probably be limited to homescreens.

**Reply by vincentcomfy** [reply]
Niagara is able to hide the status bar when on the home screen, I'm not 100% sure but I believe apps can hide the taskbar in-app as well.

**Reply by whoisabdi** [reply]
> Niagara is able to hide the status bar when on the home screen, I'm not 100% sure but I believe apps can hide the taskbar in-app as well.

Lawnchair can do that too rn wdym

**Reply by vincentcomfy** [reply]
> > Niagara is able to hide the status bar when on the home screen, I'm not 100% sure but I believe apps can hide the taskbar in-app as well.
> 
> Lawnchair can do that too rn wdym

I may have misread your comment, in my head you implied that it's not possible. My bad.

**Comment by BigCityCat** [comment]
Lawnchair does hide the status bar. But in my original comment I made the point that sometimes even if you have the option for it to be hidden it doesn't always fade in and out. Sometimes it doesn't fade in and out at all and doesn't hide itself. You would have to open and close the drawer for the fade out to begin working again. But I actually haven't noticed this behavior lately so it may have been addressed. I would have done better to file a bug report. Something I have learned to do since creating this thread. So I am going to close this. Forgot I still had it here. If I notice the behavior again i will file a bug report.
